ember-toucan-core/src/components/form/input-field.ts

```diff
@@ -0,0 +1,29 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+interface ToucanFormInputFieldComponentSignature {
+  Element: HTMLInputElement;
+  Args: {
+    error?: string;
+    label: string;
+    hint?: string;
+    isDisabled?: boolean;
+    onChange?: (value: string, e: Event | InputEvent) => void;
```

Comment:
Nit: To keep this in sync with the underlying component API you could do:

```suggestion
    // import ToucanFormInputControlComponentSignature
    onChange?: ToucanFormInputControlComponentSignature['Args']['onChange'];
```

or

```suggestion
    // import OnChangeCallback
    onChange?: OnChangeCallback<string>;
```